### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by jwPark6**
Hello! I want to merge my Korean translation of Floyd Algorithm. 
Could you confirm this?

### CODE REVIEW

File: ko/자료구조/연결 리스트/원형 연결 리스트.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## 구현
+
+- [JavaScript](https://github.com/TheAlgorithms/Javascript/blob/master/Data-Structures/Linked-List/SingleCircularLinkedList.js.js)
+- [C++](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/data_structures/cll/cll.cpp)
+- [Python](https://github.com/TheAlgorithms/Python/blob/master/data_structures/linked_list/circular_linked_list.py)
+
+## 영상 URL
+
+[Video explanation on YouTube](https://youtu.be/HMkdlu5sP4A)
```
Comment (Panquesito7) [inline ko/자료구조/연결 리스트/원형 연결 리스트.md:75]: Please translate this string into Korean (not the link, though).

File: ko/탐색/플로이드 순환 탐색 알고리즘.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+## 구현
+
+- [C++](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/search/floyd_cycle_detection_algo.cpp)
+- [C](https://github.com/TheAlgorithms/C/blob/master/searching/floyd_cycle_detection_algorithm.c)
+
+#### 영상 URL
+
+[YouTube video explaining the Floyd Cycle Detection algorithm](https://www.youtube.com/watch?v=B6smdk7pZ14)
```
Comment (Panquesito7) [inline ko/탐색/플로이드 순환 탐색 알고리즘.md:39]: Same here.

File: ko/자료구조/연결 리스트/원형 연결 리스트.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+| Deletion  |   O(1)  |  O(n) |
+
+### 실제 적용 사례
+
+- CPU 자원 할당
+- 멀티플레이어 보드 게임
+
+### SLL v.s. CLL
+
+![image](https://i0.wp.com/algorithms.tutorialhorizon.com/files/2016/03/Circular-Linked-List.png)
```
Comment (Panquesito7) [inline ko/자료구조/연결 리스트/원형 연결 리스트.md:40]:
```suggestion
![영상](https://i0.wp.com/algorithms.tutorialhorizon.com/files/2016/03/Circular-Linked-List.png)
```

File: ko/자료구조/연결 리스트/원형 연결 리스트.md (hunk 1 above)
Comment (JiHyunSong) [reply ko/자료구조/연결 리스트/원형 연결 리스트.md:75]:
```suggestion
[유투브 설명](https://youtu.be/HMkdlu5sP4A)
```

File: ko/탐색/플로이드 순환 탐색 알고리즘.md (hunk 2 above)
Comment (jwPark6) [reply ko/탐색/플로이드 순환 탐색 알고리즘.md:39]:
```suggestion
[플로이드 순환 탐색 알고리즘 유투브 영상](https://www.youtube.com/watch?v=B6smdk7pZ14)
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Panquesito7 — APPROVED** [review]

**Review by vbrazo — APPROVED** [review]

**Pull request merged**